static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.tsx

```diff
@@ -148,11 +150,23 @@ const SampleWidgetCard = styled(motion.div)<{isTable: boolean}>`
   margin: auto;
 `;
 
+const ContainerWithoutSidebar = styled('div')`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  height: 100vh;
+`;
+
 const WidgetBuilderContainer = styled('div')`
-  ${fullPageCss}
   z-index: ${p => p.theme.zIndex.widgetBuilderDrawer};
   display: flex;
   align-items: center;
   justify-content: space-between;
   height: 100vh;
+  position: fixed;
+  width: -webkit-fill-available; /* Chrome */
+  width: -moz-available; /* Firefox */
+  width: fill-available; /* others */
```

Comment:
don't mind the jankiness of this 😭  it works for now and for demos but need to check out if there's a better way to do this